Source/JavaScriptCore/b3/B3ValueRep.cpp

```diff
@@ -94,6 +94,13 @@ void ValueRep::dump(PrintStream& out) const
     RELEASE_ASSERT_NOT_REACHED();
 }
 
+// We use `B3::ValueRep` for bookkeeping in the BBQ wasm backend, including on
+// 32-bit platforms, but not for code generation (yet!), so we don't actually
```

Comment:
If you remove this, does it really not compile? Maybe you can add stubs for the needed instructions instead for now?